cartographer/mapping/sparse_pose_graph.cc

```diff
@@ -79,16 +79,14 @@ proto::SparsePoseGraph SparsePoseGraph::ToProto() {
     for (size_t old_node_index = 0;
          old_node_index != single_trajectory_nodes.size(); ++old_node_index) {
       const auto& node = single_trajectory_nodes[old_node_index];
-      if (!node.trimmed()) {
```

Comment:
This is not possible until `GetTrajectoryNodes()` returns a `MapById<>` instead of `vector<vector<>>`?

Reply:
Ok. I will clean this up then when I change to MapById.